chromium_src/extensions/browser/sandboxed_unpacker.cc

```diff
@@ -41,10 +41,8 @@ bool BraveRewriteManifest(const std::string& extension_id,
 
 }  // namespace brave
 
-#define BRAVE_SANDBOXEDUNPACKER_REWRITEMANIFESTFILE              \
-  base::Value::Dict* dict_manifest = final_manifest.GetIfDict(); \
-  DCHECK(dict_manifest);                                         \
-  brave::BraveRewriteManifest(extension_id_, dict_manifest);
+#define BRAVE_SANDBOXEDUNPACKER_REWRITEMANIFESTFILE \
+  brave::BraveRewriteManifest(extension_id_, &final_manifest);
```

Comment:
is this even relevant anymore?

Reply:
@yrliou Do you happen to know if we still need to rewrite the manifest to add socket permissions for the IPFS Companion extensions?

Reply:
@emerick I also pinged @cypt4

Reply:
No longer needed, removed here 1893df15a2a1f4ef4cd3864cbc158341415189a7.